overrides/main.html

```diff
@@ -12,12 +12,12 @@
   <meta name="twitter:creator" content="@Panel_org" />
   <meta name="twitter:title" content="{{ title }}" />
   <meta name="twitter:description" content="{{ config.site_description }}" />
-  <meta name="twitter:image" content="https://holoviz-topics.github.io/panel-chat-examples/assets/images/panel-chat-examples-card-800x418.png" />
```

Comment:
changing the `version` will trigger twitter to fetch the new image and update its cache.